config/admin.php

```diff
@@ -74,6 +74,41 @@ function clean_dashboard(){
 
 /**
  *******************************************************************************
+ * change logo link from wordpress.org to user site
+ *******************************************************************************
+ *
+ * function changes url of the admin logo
+ *
+**/
+
+function login_url () {  return home_url(); }
```

Comment:
Please don't put the function definitions on one line to keep it consistant with the rest of the codebase. This should read:
```
function login_url () {  
    return home_url(); 
}
```

Reply:
Will do!